pkg/envoy/lds/inmesh.go

```diff
@@ -88,6 +92,14 @@ func (lb *listenerBuilder) getInboundHTTPFilters(trafficMatch *trafficpolicy.Tra
 		}
 		filters = append(filters, rateLimitFilter)
 	}
+	// Apply the network level global rate limit filter if configured for the TrafficMatch
+	if trafficMatch.RateLimit != nil && trafficMatch.RateLimit.Global != nil && trafficMatch.RateLimit.Global.TCP != nil {
+		rateLimitFilter, err := buildTCPGlobalRateLimitFilter(trafficMatch.RateLimit.Global.TCP, trafficMatch.Name)
```

Comment:
this is all in the func `getInboundHTTPFilters`, should this be using an HTTP Global rate limit? Or will you eventually be checking both TCP and HTTP Rate limit existance here?

If the latter, is it possible to specify both on a single traffic match?

Reply:
Yes, the TCP rate limiting is applicable to HTTP traffic, so much like local TCP rate limiting filter, the global TCP rate limiting filter can be applied on the HTTP filter_chain as well. Think of it as performing connection level rate limiting at the TCP level, for HTTP traffic.